CHANGES.md

```diff
@@ -4,7 +4,8 @@
 - [NEW] Moved `create_query_index` and other query related methods to `CouchDatabase` as the `_index`/`_find` API is available in CouchDB 2.x.
 - [NEW] Support IAM authentication in replication documents.
 - [IMPROVED] Added support for IAM API key in `cloudant_bluemix` method.
-- [IMPROVED] Updated Travis CI and unit tests to run against CouchDB 2.1.1.
+- [IMPROVED] Updated Travis CI and unit tests to run against CouchDB 1.7.1 and 2.1.1.
+- [IMPROVED] Updated Python environment for testing in Travis from 3.5 to 3.6.
```

Comment:
I actually don't think we need these entries as they are related to the testing not the library release.
That said if we're saying that Python 3.6 is supported it may be worth saying something like:
`- [IMPROVED] Verified library operation on Python 3.6.`(we should say which 3.6.x version)